### PR TITLE
Add wkw param control for stress testing wkws + update doc

### PR DESCRIPTION
**Pull request by mbenguig**

### CODE REVIEW

File: Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml [hunk 1]
```diff
@@ -140,49 +145,42 @@ variables.put("waitForSignals", True)
...
-def total_nb_steps = stressed_risk_free_rate_steps * stressed_volatility_steps
+def total_nb_steps = stressed_risk_free_rate_range_nb_steps * stressed_volatility_range_nb_steps
 def nb_steps_per_task = total_nb_steps / nb_replicated_tasks
 if (total_nb_steps % nb_replicated_tasks != 0)
 	throw new Exception("total_nb_steps % nb_replicated_tasks != 0")
 
 // Compute ranges
-def stressed_risk_free_rate_step_size_in_percent = (stressed_risk_free_rate_max_in_percent - stressed_risk_free_rate_min_in_percent) / (stressed_risk_free_rate_steps - 1)
-def stressed_volatility_step_size_in_percent = (stressed_volatility_max_in_percent - stressed_volatility_min_in_percent) / (stressed_volatility_steps - 1)
+def stressed_risk_free_rate_step_size_in_percent = (stressed_risk_free_rate_in_percent_range_max - stressed_risk_free_rate_in_percent_range_min) / (stressed_risk_free_rate_range_nb_steps - 1)
```
Comment (fviale) [inline Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml:167]: what happens when stressed_risk_free_rate_range_nb_steps == 1 ?

File: Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml [hunk 2]
```diff
@@ -140,49 +145,42 @@ variables.put("waitForSignals", True)
...
+def stressed_volatility_step_size_in_percent = (stressed_volatility_in_percent_range_max - stressed_volatility_in_percent_range_min) / (stressed_volatility_range_nb_steps - 1)
```
Comment (fviale) [inline Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml:168]: same comment as above regarding possible division by zero

File: Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml [hunk 3]
```diff
@@ -1,19 +1,24 @@
...
+    <variable name="stressed_volatility_in_percent_range_max" value="+5" model="PA:INTEGER" description="The max of the stressed volatility rate range." group="Simulation" advanced="false" />
+    <variable name="stressed_volatility_range_nb_steps" value="8" model="PA:INTEGER" description="The step number of the stressed volatility rate range." group="Simulation" advanced="false" />
+    <variable name="nb_replicated_tasks" value="4" model="PA:INTEGER" description="The number of ProActive replicated tasks which must divide stressed_risk_free_rate_range_nb_steps x stressed_volatility_range_nb_steps." group="Parallelization" advanced="false" />
+    <variable name="PARAMS_CHECK" value="" model="PA:SPEL((T(Integer).parseInt(variables[&#x27;stressed_risk_free_rate_range_nb_steps&#x27;]) * T(Integer).parseInt(variables[&#x27;stressed_volatility_range_nb_steps&#x27;])) % T(Integer).parseInt(variables[&#x27;nb_replicated_tasks&#x27;]) == 0 )" description="Check (stressed_risk_free_rate_range_nb_steps x stressed_volatility_range_nb_steps) % nb_replicated_tasks == 0"  advanced="false" hidden="true"/>
```
Comment (fviale) [inline Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml:21]: nice SPEL expression :-)

File: Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml [hunk 4]
```diff
@@ -1,19 +1,24 @@
...
+    <variable name="assets_params_file_path_from_dataspace" value="assets.txt" model="PA:GLOBAL_FILE" description="The relative path from the dataspace of the asset params file following the csv format." group="Simulation" advanced="false" />
+    <variable name="assets_correlations_file_path_from_dataspace" value="assets_correlations.txt" model="PA:GLOBAL_FILE" description="The relative path from the dataspace of the asset correlations file." group="Simulation" advanced="false" />
+    <variable name="stressed_risk_free_rate_in_percent_range_min" value="-10" model="PA:INTEGER" description="The min of the stressed risk free rate range." group="Simulation" advanced="false" />
+    <variable name="stressed_risk_free_rate_in_percent_range_max" value="+10" model="PA:INTEGER" description="The max of the stressed risk free rate range." group="Simulation" advanced="false" />
+    <variable name="stressed_risk_free_rate_range_nb_steps" value="8" model="PA:INTEGER" description="The step number of the stressed risk free rate range." group="Simulation" advanced="false" />
```
Comment (fviale) [inline Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml:16]: to remove the risk of division by zero you could use the notation PA:INTEGER[2,1000] (for example)

File: Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio.xml [hunk 5]
```diff
@@ -3,14 +3,19 @@
...
+    <variable name="assets_correlations_file_path_from_dataspace" value="assets_correlations.txt" model="PA:GLOBAL_FILE" description="The relative path from the dataspace of the asset correlations file." group="Simulation" advanced="false" />
+    <variable name="stressed_risk_free_rate_in_percent_range_min" value="-10" model="PA:INTEGER" description="The min of the stressed risk free rate range." group="Simulation" advanced="false" />
+    <variable name="stressed_risk_free_rate_in_percent_range_max" value="+10" model="PA:INTEGER" description="The max of the stressed risk free rate range." group="Simulation" advanced="false" />
+    <variable name="stressed_risk_free_rate_range_nb_steps" value="8" model="PA:INTEGER" description="The step number of the stressed risk free rate range." group="Simulation" advanced="false" />
+    <variable name="stressed_volatility_in_percent_range_min" value="-5" model="PA:INTEGER" description="The min of the stressed volatility rate range." group="Simulation" advanced="false" />
```
[the remark this answers is not in the document]
Comment (mbenguig) [reply Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio.xml:14]: hummmm  .. no!

File: Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml (hunk 3 above)
Comment (mbenguig) [reply Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml:21]: Thx. (your padawan)

File: Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio.xml (hunk 5 above)
[the remark this answers is not in the document]
Comment (fviale) [reply Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio.xml:14]: a metter of taste I agree

File: Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml (hunk 1 above)
Comment (mbenguig) [reply Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml:167]: It throws a div by 0 error ! (thx)

File: Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml (hunk 4 above)
Comment (mbenguig) [reply Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml:16]: ok thx!

Comment (mbenguig) [reply Finance/resources/catalog/Stress_Testing_Monte_Carlo_Value_Portfolio_MC_Interactive_Control.xml]: PR updated.

**Pull request merged**